.github/workflows/antsibull-docs.yml

```diff
@@ -49,7 +49,8 @@ jobs:
           python3 -m venv venv
           . ./venv/bin/activate
           python -m pip install --upgrade pip
-          python -m pip install . ../antsibull-docs "pyre-check ~= 0.9.15"
+          python -m pip install ../antsibull-core
+          python -m pip install . "pyre-check ~= 0.9"
```

Comment:
```suggestion
          python -m pip install . ../antsibull-core "pyre-check >= 0.9"
```

Reply:
That obviously doesn't work. It installes antsibull-core from PyPI instead of using the one from ../antsibull-core.

Reply:
(Otherwise https://github.com/ansible-community/antsibull-core/pull/54#issuecomment-1510361852 would not have happened.)

Reply:
I don't think the problem is the single transaction. I made a typo: `.` and `../antsibull-docs` are the same directory. It should've been `.` (antsibull-docs is cwd) and `../antsibull-core`.

Reply:
Hmm, I thought the typo was in the other file and thus not relevant here. Let's try again...

Reply:
Yes, seems good now. It fails (as it should).

Reply:
Yeah :). Running a single transaction should be better, as it makes sure that the resolver doesn't replace `../antsibull-core` installed in the first transaction with `antsibull-core` from PyPI.

Reply:
Do you also want to change `pyre-check ~= 0.9` to `pyre-check >= 0.9` or not in this PR?

Reply:
Hmm, I would stick to `~= 0.9` (and not `~= 0.9.17`) since that's what the old antsibull(-docs) version was also using.